tensornetwork/linalg/initialization.py

```diff
@@ -200,3 +201,7 @@ def random_uniform(shape: Sequence[int],
   the_tensor = initialize_tensor("random_uniform", shape, backend=backend,
                                  seed=seed, boundaries=boundaries, dtype=dtype)
   return the_tensor
+def initialize_orthogonal_tensor_wrt_pivot(shape=Sequence[int],dtype:Optional[Type[np.number]]=None,pivot_axis:int=-1,seed=Optional[int]=None,backend: Optional[Union[Text, AbstractBackend]] = None,non_negative_diagonal:bool=False) ->Tensor:
```

Comment:
I'm wondering if we could find a less clunky name. Some possibilities that come to my mind are random_orthogonal or random_isometry @alewis?

Reply:
Pls add a docstring that explains what the function is doing, what the arguments are, and what the returned values are.